deployment/deploy/deploy.cpp

```diff
@@ -1135,6 +1151,9 @@ IPropertyTree* getInstances(const IPropertyTree* pEnvRoot, const char* compName,
       const char* buildSet= pComponent->queryProp("@buildSet");
       const char* logDir = NULL;
 
+      if ((!strcmp(buildSet,"thor") || !strcmp(buildSet,"roxie")) && !pTopologyComponents.contains(name))
```

Comment:
Why do you check for thor and roxie here but use Software/Topology/* to add to pTopologyComponents?